api.php

```diff
@@ -164,28 +184,53 @@ function listMod($modid)
 		);
 	}
 
+	$srows = $con->getAll("
+		select 
+			fileid,
+			assetid,
+			filename,
+			thumbnailfilename,
+			created
+		from 
+			`file` 
+		where assetid=?
+	", array($modid));
+
+	$screenshots = array();
+	foreach ($srows as $screenshot) {
+		$screenshots[] = array(
+			"fileid" => intval($screenshot["fileid"]),
+			"mainfile" => "files/asset/{$screenshot["assetid"]}/" . $screenshot["filename"],
+			"filename" => $screenshot["filename"],
+			"thumbnailfilename" => $screenshot["thumbnailfilename"],
+			"created" => $screenshot["created"]
+		);
+	}
+
 	$mod = array(
 		"modid" => intval($row["modid"]),
 		"assetid" => intval($row["assetid"]),
 		"name" => $row['name'],
 		"text" => $row['text'],
 		"author" => $row['author'],
 		"urlalias" => $row['urlalias'],
-		"logofilename" => $row['logofilename'] ? "asset/{$row['assetid']}/" . $row['logofilename'] : null,
+		"logo" => $row['logofilename'] ? "files/asset/{$row['assetid']}/" . $row['logofilename'] : null,

```

Comment:
the `files/` prefix is not necessary - by definition all file paths need to use the base url by convention as documented here: https://github.com/anegostudios/vsmoddb#urls

Reply:
alternatively we change the convetion in the docs

Reply:
I've missed the convention, maybe removing the `files/` prefix from `/api/mods` endpoint is a better change